src/pages/Patient/components/Patient__List.res

```diff
@@ -16,13 +13,13 @@ type state = {
 
 let getStatusType = (status: string) => {
   switch status {
-  | "POSITIVE" => POSITIVE
-  | "SUSPECTED" => SUSPECTED
-  |_ => SUSPECTED
+  | "POSITIVE" =>Patient__Types.POSITIVE
+  | "SUSPECTED" => Patient__Types.SUSPECTED
+  |_ => Patient__Types.SUSPECTED
   }
 }
```

Comment:
Hey so can we move this function to the types folder itself? Right now, we are referencing `Patient__Types` at 6 place, this will reduce it to 2. since you are using the exact same function in both the files... it's better to move it to the types file, right?